rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/directives/QueueDirectives.scala

```diff
@@ -76,4 +77,18 @@ trait QueueDirectives {
       body(queueData)
     }
   }
+
+  private def checkOnlyQueueNameInUri(): Directive1[String] = {
+    extractUri.flatMap {
+      uri => {
+        val onlyQueueNameInUriPath = Option(uri.path.toString().tail)
```

Comment:
doesn't this always return `Some(the uri without the first character)`?

Reply:
Thanks for advice, now I'm checking if there is only one segment in uri.